content/en/docs/started/installing-kubeflow/release-info/latest.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
We might need to add information on how to update those JSONs into release Handbook: https://github.com/kubeflow/community/blob/master/releases/handbook.md
cc @kubeflow/release-team 

@thesuperzapper Would it be easier to have single JSON with the list of Kubeflow versions and publish dates ?

Reply:
@andreyvelich you can add that if you like, but the system is designed to fail the website build when a distribution tries to claim a version that it does not know about.

The error will tell the person making that PR to run the script at [`./content/en/docs/started/installing-kubeflow/get_new_releases.sh`](https://github.com/kubeflow/website/blob/91fa5f5a53df057166c1bee5733914d49a2f216e/content/en/docs/started/installing-kubeflow/get_new_releases.sh).

That script uses the [releases on the `kubeflow/manfistes`](https://github.com/kubeflow/manifests/releases) repo to get the versions and determine the release dates.

Reply:
But you have reminded me to run the script on this PR as we just released 1.9.1, I will do that now.